### PR TITLE
Herck/demos ic and drug

### PR DESCRIPTION
**Pull request by IlsevanHerck**
Add demos for initial conditions and drug effects

Addresses #89

## Proposed Changes

  - Add demo for use of initial conditions and state
  - Add demo for use of drug scaling factors
  -

### CODE REVIEW

**Review by cdaversin — COMMENTED** [review]
This PR looks good to me, we can merge once the mypy error is fixed (`demos/drug_effects_demo.py:33: error: Argument "drug_factors_file" to "Config" has incompatible type "Path"; expected "str"`), and the doc generation error (not sure why it cannot find new demo though, (@finsberg, any idea ?)

File: demos/drug_effects_demo.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+# This will set :
+#
+# ```
+# {'PCL': 1000,
+#  'T': 1000,
+#  'bnd_rigid': False,
+#  'cell_init_file': '',
+#  'disease_state': 'healthy',
+#  'drug_factors_file': '',
```
[the remark this answers is not in the document]
Comment (IlsevanHerck) [reply demos/drug_effects_demo.py:45]: Good catch, it was added in the wrong line (population file). 
I agree that only showing it in the simple_demo and printing it here should be enough.

File: demos/drug_effects_demo.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+#
+# We can also save the output to xdmf-files that can be viewed in Paraview
+#
+
+simcardems.postprocess.make_xdmffiles(outdir.joinpath("results.h5"))
+
+#
+# The `xdmf` files are can be opened in [Paraview](https://www.paraview.org/download/) to visualize the different variables such as in {numref}`Figure {number} <drug-demo-paraview>`.
+#
```
Comment (finsberg) [inline demos/drug_effects_demo.py:80]: @IlsevanHerck Did you forget to push this figure or should you refer to the state traces?

Comment (IlsevanHerck) [reply demos/drug_effects_demo.py]: I meant to remove that example, now fixed in [https://github.com/ComputationalPhysiology/simcardems/pull/108/commits/990e09669cd1ea5da7b0738a93c90e7093f03b4f](url)

**Pull request merged**